stephan-livera-podcast/2019-01-20-nicolas-dorier-and-btcpayserver.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Nicolas Dorier and BTCPayServer – self hosted Bitcoin and Lightning payments 
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['Nicolas Dorie']
+date: 2019-01-20
+media: https://stephanlivera.com/episode/48/
+---
+media: https://stephanlivera.com/download-episode/955/48.mp3
```

Comment:
This should be the media link in the front matter and not repeated here

Reply:
I changed the download link to media and the episode link to podcast.